scaling_performance/scaling_cluster_monitoring.adoc

```diff
@@ -0,0 +1,131 @@
+[[scaling-performance-cluster-monitoring]]
+= Scaling Cluster Monitoring Operator
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+== Overview
+
+{product-title} exposes metrics that can be collected and stored in back-ends by
+the link:https://github.com/openshift/cluster-monitoring-operator[cluster-monitoring-operator]
+As an {product-title} administrator, you can view system resources, containers and components metrics in
+one dashboard interface (Grafana).
+
+This topic provides information on scaling the cluster monitoring operator.
+
+
+[[cluster-monitoring-recommendations-for-OCP]]
+== Recommendations for {product-title}
+* Use at least 3 infra nodes as best practice (for 2 Prometheus pods and operator pods)
+xref:../admin_guide/manage_nodes.adoc#infrastructure-nodes[infrastructure
+nodes].
+* Use at least 3 OCS nodes (with NVMe drives) link:https://www.redhat.com/de/technologies/cloud-computing/openshift-container-storage[*openshift-container-storage*].
```

Comment:
three OCS nodes

What does OCS stand for? If it's an acronym, we should define that here too. Also, this line currently does not read like a full sentence, as the linked text seems out of place. Is a word missing?

Reply:
it's acronym right, openshift container storage as Elko suggest us to use it instead of glusterfs